moz_generic/tests/hindsight/run/output/d2p.cfg

```diff
@@ -0,0 +1,29 @@
+filename        = "s3_parquet.lua"
+message_matcher = "Type == 'moz_generic' && Fields[namespace] == 'generic' && Fields[docType] == 'test' && Fields[docVersion] == '1'"
+ticker_interval = 60
+preserve_data   = true
+
+parquet_schema = [=[
```

Comment:
I've removed this test. It was useful for me when developing, but it indeed does not really test anything new.

As for a generic_s3_parquet output, that was the plan, but per https://bugzilla.mozilla.org/show_bug.cgi?id=1368197#c4 I think it's simpler to dynamically generate the hindsight configs and use the existing output than to write a multiplexed parquet output.